app/models/user.rb

```diff
@@ -61,6 +63,12 @@ def trust_level
     community_user.trust_level
   end
 
+  # Checks whether this user is the same as a given user
+  # @param [User] user user to compare with
+  def same?(user)
```

Comment:
I'd rename this `same_as?` - it reads slightly more naturally when the method is used. Not exactly a major issue, though.

Reply:
Agreed, sounds better - done